tests/test_docs.py

```diff
@@ -143,32 +143,32 @@ def testChekListParamsGoogledoc(self):
         doc = googledocs
         d = docs.DocString(myelem, '    ', doc)
         d._extract_docs_params()
-        self.failUnless(d.get_input_style() == 'google')
+        self.assertTrue(d.get_input_style() == 'google')
 
     def testAutoInputStyleGoogledoc(self):
         doc = googledocs
         d = docs.DocString(myelem, '    ', doc)
-        self.failUnless(d.get_input_style() == 'google')
+        self.assertTrue(d.get_input_style() == 'google')
 
-    def testAutoInputStyleNumpydoc(self):
+    def testAutoInputSyleNumpydoc(self):
```

Comment:
There is a type on `Style` word

Reply:
Oops. I'll fix that accident... well spotted!